web/crux-ui/src/components/products/edit-product-card.tsx

```diff
@@ -143,6 +145,7 @@ const EditProductCard = (props: EditProductCardProps) => {
           <DyoButton className="hidden" type="submit"></DyoButton>
         </form>
       </DyoCard>
+      <ProductVersionsSection productId={product.id} versions={props.versions} readonly />
```

Comment:
In my opinion `ProductVersionsSection` shouldn't handle the undefined case. In `ProductVersionsSection` `versions` should be mandatory and `EditProductCard` should hande the undefined case (for example `props.version ?? []`).

This way `ProductVersionsSection` doesn't need an empty fragment.